tests/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ if (OE_SGX)
   set(DEFINE_OE_SGX "-DOE_SGX")
 endif ()
 
+add_subdirectory(out_of_proc_quote_generation)
```

Comment:
Any reason putting this separately from the list below?

Reply:
Actually no, probably at the moment I added a new line, I was thinking maybe there was some relationship among the list below.

Reply:
Moved into the `UNIX`-`OE_SGX` branches.